modules/RTC/JitsiLocalTrack.js

```diff
@@ -863,15 +863,14 @@ export default class JitsiLocalTrack extends JitsiTrack {
 
         this._setEffectInProgress = true;
 
-        // TODO: Create new JingleSessionPC method for replacing a stream in JitsiLocalTrack without offer answer.
-        return conference.removeTrack(this)
+        return conference._removeLocalTrackAsMute(this)
```

Comment:
This is not ideal to mix mute operation into the effects flow. What does need to happen for the effect operation? Can we add a new flow for the effect?

Reply:
Like discussed, I am going to rename the remove/add methods to something generic since both the mute/unmute operation and effect have the exact same flow, i.e., remove or add the track from/to RTCPeerConnection but not from/to the TraceablePeerConnection.